### PR TITLE
Pipebomb

### PR DESCRIPTION
**Pull request by Wouju**
## About The Pull Request

I noticed a lack of even a IED for vagabonds to craft, so  have a pipebomb

image
image

image
when shoved in mouth (300-400 dmg)

image
image
at the above distance

image
image
image
bit hard to hide underneath backpacks and satchels

## Why It's Good For The Game

Pipebombs are an important part of cultures all around the world. Being able to make homemade explosives and suffer severe injuries from mishandling of chemicals is a generations long tradition. Also pipebomb in your disposal bin.

## Changelog
:cl:
add: Added a craftable pipebomb
/:cl:

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
Needs more igniters.

**Review by SirRichardFrancis — COMMENTED** [review]
I like the idea. However, those are cheap and could be mass produced (can craft 9+ of those simultaneously, 15 steps is not that big of a deal) so consider reducing amount of shrapnel, 150 is a lot.
Maybe ask for a sprite in spriters channel instead of shitpost next time.

**Comment by ValoTheValo** [comment]
Thier is a pipe bomb in your delivery chute

**Comment by Wouju** [comment]
> I like the idea. However, those are cheap and could be mass produced (can craft 9 of those simultaneously, so 15 steps is not that big of a deal) so consider reducing amount of shrapnel, 150 is a lot.

True, im going to do more testing when i get home

**Comment by ChungusGamer666** [comment]
Ted kaczynski speedrun 100% WR

**Comment by Jan-Blasiak** [comment]
Is the intraship mailing system working properly already?

**Comment by Kurgis** [comment]
> Is the intraship mailing system working properly already?
No, a map change around ~2020 broke it due to not setting delivery tags.

**Comment by TheShown911** [comment]
Jesus Christ Denton a bomb.

File: code/game/objects/items/weapons/grenades/frag.dm [hunk 1]
```diff
@@ -43,3 +43,13 @@
 	matter = list(MATERIAL_BIOMATTER = 75)
 	fragment_damage = 7
 	damage_step = 3
+
+/obj/item/grenade/frag/pipebomb
+	name = "Improvised Pipebomb"
```
Comment (Firefox13) [inline code/game/objects/items/weapons/grenades/frag.dm:48]:
```suggestion
	name = "improvised pipebomb"
```

**Review by TheShown911 — COMMENTED** [review]
But... it's not a pipe...

**Comment by Wouju** [comment]
> But... it's not a pipe...

Eris is a roleplaying server, as such just use your imagination

**Review by Firefox13 — APPROVED** [review]
there's a PR in your mailbox

**Pull request merged**